HW-3/task2.js

```diff
@@ -0,0 +1,21 @@
+/*Task2*. Решить квадратные уравнения. Переменные называть по правилам.
+Вывести в консоль ответы в виде "Ответ к уравнению 1: <корень>", "Ответ к уравнению 2: <корень> и <корень>"
+1.  x2 - 6x + 9 = 0. - один корень
+2.  x2 - 4x - 5 = 0. - два корня
+*/
+
+//Уравнение с одним корнем:
+const aFirstTerm1=1;
+const bSecondTerm1=-6;
+const cFreeTerm1=9;
```

Comment:
If I'm not mistaken, check the spelling of this variable `cFreeTerm1` and similar to it

Reply:
It depends... Mostly they are colled just "terms' abroad but I found such variation as 'quadratic term' for a, 'linear term' for b and 'constant term' for c. I can use them.